lib/index.js

```diff
@@ -102,8 +102,8 @@ internals.methods = {
         internals.assert(value, schema, true, args);
     },
 
-    attempt(value, schema, ...args /* [message], [options] */) {
-
+    attempt(value, schema, message = '', options) {
+        const args = [message, options];
```

Comment:
Is this change really needed now?
The `null` check on L223 must be enough, right?

Reply:
it is indeed, I already changed it back with othes small corrections in code patterns a have break